pinot-core/pom.xml

```diff
@@ -218,7 +218,7 @@
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
+      <artifactId>mockito-inline</artifactId>
```

Comment:
was there something you needed from this that core couldn't achieve?

Reply:
Test static function. Some other module also replace core with inline. If I remember correctly, pinot-common has a detailed comment in the pom